src/views/articles/subcategory/SingleArticle.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Thought you suggested that this should be deleted since there is already an component for displaying an article